src/_pytest/config/__init__.py

```diff
@@ -828,7 +828,7 @@ def _import_plugin_specs(
     ) -> None:
         plugins = _get_plugin_specs_as_list(spec)
         for import_spec in plugins:
-            self.import_plugin(import_spec)
+            self.import_plugin(import_spec, consider_entry_points=True)
```

Comment:
Please add tests to prove that it works and demonstrate the difference with the previous behavior.

Reply:
I can probably start with some merge of `test_missing_required_plugins` (it looks like the `pytest11` entry point is created there) and `test_early_config_cmdline` (some kind of `PYTEST_PLUGINS` handling).  But then some questions arises, for example:  why is there the `pytester.syspathinsert()` call?  After reading [the doc](https://docs.pytest.org/en/8.2.x/reference/reference.html#pytest.Pytester.syspathinsert) it looks like the call is just a no-op, but this conclusion is apparently wrong.

I'm very sorry, but I'll need to leave this for somebody else who actually knows Python.  Disclaimer: I'm not a Python developer.

Reply:
I'd imagine that it sets `$PYTHONPATH` just for the subprocess invocation of pytest, which is what makes it possible for that subprocess to `import <tested_pytest_plugin>` residing on that path.

Reply:
I wouldn't merge tests at this point. Maybe, in a separate refactoring. But it's always easier to add a separate one and not care if you break other ones accidentally. Especially, if you're unsure.

As for the tips, using `monkeypatch.setenv()` should be a way to test the env var bit. It might be a bit more involved with generating a custom config, though. Still, I think it's a good starting point.

Reply:
Sorry, this is really out of my knowledge.

Reply:
Okay, so don't mark it as resolved because hiding the discussion from people doesn't mean that it shouldn't be solved before merging.

Reply:
Looks like the right place to put the tests in would be `test_pluginmanager.py::TestPytestPluginManager`

Reply:
There's also `test_early_load_setuptools_name` @ `acceptance_test.py`